include/SFML/System/Angle.hpp

```diff
@@ -138,26 +138,32 @@ class Angle
     ////////////////////////////////////////////////////////////
     // NOLINTNEXTLINE(readability-identifier-naming)
     static const Angle Zero; //!< Predefined 0 degree angle value
+    // NOLINTNEXTLINE(readability-identifier-naming)
+    static const Angle QuarterCircle; //!< Predefined 90 degree angle value
+    // NOLINTNEXTLINE(readability-identifier-naming)
+    static const Angle HalfCircle; //!< Predefined 180 degree angle value
+    // NOLINTNEXTLINE(readability-identifier-naming)
+    static const Angle FullCircle; //!< Predefined 360 degree angle value
```

Comment:
Please always highlight new APIs in the PR description, at best keep them as separate commit or even PR.

Reply:
I'd also rather save such additions for a later PR as to keep this PR as minimal as possible

Reply:
OK, will remove for a later PR, I think these are beneficial to avoid using `degrees` and relying on compile-time conversions that might not be done at compile-time